.eslintrc.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+    'env': {
+        'browser': true,
+        'commonjs': true,
+        'es6': true
+    },
+    'parser': 'babel-eslint',
+    'extends': [
+        'eslint:recommended',
+        'plugin:react/recommended'
+    ],
+    'parserOptions': {
+        'ecmaFeatures': {
+            'experimentalObjectRestSpread': true,
+            'jsx': true,
+            'modules': true
+        },
+        'sourceType': 'module'
+    },
+    'plugins': [
+        'react'
+    ],
+    'rules': {
+        'indent': [
+            'error',
+            4
```

Comment:
I prefer 2! But let it be ;-)

Reply:
I prefer 4 :) but we can agree on any setting!